lib/travis/sso/generic.rb

```diff
@@ -102,7 +101,7 @@ def allow(request)
         def login(request)
           return unless token = sso_token(request)
           raw = if enterprise
-            open("#{endpoint}/users?access_token=#{token}", 'Accept' => accept, 'ssl_ca_cert' => ssl_ca_cert )
+            open("#{endpoint}/users?access_token=#{token}", 'Accept' => accept, 'ssl_verify_mode' => OpenSSL::SSL::VERIFY_NONE )
```

Comment:
:/ if you really want to go this path, there should be an option like 'yes_i_really_want_to_turn_off_cert_verification' , not 'if enterprise'